subsys/bluetooth/controller/hci_driver.c

```diff
@@ -439,22 +399,42 @@ static int ble_enable(void)
 	return 0;
 }
 
-static int hci_driver_init(struct device *unused)
+static const struct bt_hci_driver drv = {
+	.name = "Controller",
+	.bus = BT_HCI_DRIVER_BUS_VIRTUAL,
+	.open = hci_driver_open,
+	.send = hci_driver_send,
+};
+
+u8_t bt_read_static_addr(bt_addr_le_t *addr)
```

Comment:
@rugeGerritsen You might want to pay attention to the upmerge PR about this change.

The function has been changed to this: `uint8_t bt_read_static_addr(struct bt_hci_vs_static_addr *addr)`
This is has been done to avoid having to store the identity in flash. As the identity can be read from FICR with the IR to get a valid privacy identity.

Reply:
Thanks for pointing out. We can probably wait with merging this until the upmerge is done

Reply:
Sorry. Poorly phrased by me. You should pay attention to the bt_read_static_addr function in the upmerge, since it is changed there.